### ISSUE TITLE
Description of backbone taxonomy missing list of sources

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#585]
**Description of backbone taxonomy missing list of sources**

At the end of the description we refer to the 'following 54 sources' but they are not currently listed.

-----
fbitem-datasetd7dddbf4-2cf0-4f39-9b2a-bb099caae36c
User provided contact info: timhirsch
System: Chrome 61.0.3163 / Mac OS X 10.12.6
Referer: https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c
Window size: width 1600 - height 784
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-10-26T05:36:50.983Z',mode:absolute,to:'2017-10-26T05:42:50.983Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-10-26T05:36:50.983Z',mode:absolute,to:'2017-10-26T05:42:50.983Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by mdoering** [comment]
Yes, thats blocked by gbif/registry#27

**Comment by thomasstjerne** [comment]
List of constituent datasets is now added to the backbone dataset page through https://github.com/gbif/portal16/commit/eb9984f824787b28889b3d8b9e57d00a1aad1ceb

**Comment by mdoering** [comment]
The <ul> list is removed even if the 2 <para> sections are merged into a single paragraph

**Comment by mdoering** [comment]
Also `<br/>` tags are removed if within the para tag. I have updated the backbone metadata with the [attached EML](https://github.com/gbif/portal-feedback/files/1478388/nub-eml.txt) delimiting sources by a plain semicolon now.
